src/components/TopBar/TopBarComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import UploadIcon from 'components/TopBar/images/icon-uploads.svg';
+import SearchIcon from 'components/TopBar/images/icon-search.svg';
+import MenuIcon from 'components/TopBar/images/icon-menu.svg';
+import {
+  StyledControls,
+  StyledIcon,
+  StyledHeader,
+  StyledTopBar,
+  StyledTopBarWrapper,
+  BottomLine,
+} from 'components/TopBar/TopBarStyles';
+
+export class TopBarComponent extends Component {
+  render() {
+    return (
+      <StyledTopBarWrapper>
+        <StyledTopBar>
+          <StyledHeader>NeuralNotes</StyledHeader>
```

Comment:
I don't think "header" is the best name here.
Usually a header is something like the top bar :) but here it's just a text that is displayed in a column. It's not even a row :)
[In Bootstrap it's called "brand"](https://getbootstrap.com/docs/4.0/components/navbar/): 
![image](https://user-images.githubusercontent.com/6686679/83453288-e2f9e680-a462-11ea-90e7-69b553279469.png)

Also it may be called "logo". Or "app name" :)

![image](https://user-images.githubusercontent.com/6686679/83453490-42f08d00-a463-11ea-9c4a-8959e592448f.png)

Reply:
`TopBar` is a header in fact, by html5 semantic markup :)

![image](https://user-images.githubusercontent.com/6686679/83453760-bc887b00-a463-11ea-80ef-1601fec73775.png)

And our thing is called `Logo` there :)

https://www.semrush.com/blog/semantic-html5-guide/